### ISSUE TITLE
Images in step 1 of trajectory tutorial don't match the paragraphs

### ISSUE THREAD

**Issue by calcmogul** [wpilibsuite/frc-docs#1099]
Some images show gains for a position control loop, but the docs only talk about velocity control. Furthermore, the explanation of the velocity control gain doesn't use the number in the image.

https://docs.wpilib.org/en/stable/docs/software/examples-tutorials/trajectory-tutorial/characterizing-drive.html

**Comment by jasondaming** [comment]
Yeah I think this is one of the things I remember when I created #953.  I think the whole tutorial needs an update.  I planned to do it earlier and had all the right stuff but the drivetrain I was using was Pneumatic and threw all the numbers off significantly from a KOP drivetrain.

**Comment by calcmogul** [comment]
If all else fails, we could recreate the screenshots and stuff with a simulated drivetrain.

**Comment by jasondaming** [comment]
Joe Ross mentioned he did that and it helped his simulation, so I was planning to do it soon.  Do you think the numbers would be close to what users would see with a kitbot?

**Comment by jasondaming** [comment]
I would like to help with this but I don't this the numbers I got from frc-char in simulation would make a good example here with the P value being so low and the trackwidth being insanely high.
![image](https://user-images.githubusercontent.com/15695521/105213238-141f4800-5b14-11eb-8b91-109e62120a5c.png)

**Comment by Daltz333** [comment]
Step 2 also has some innacurravies.
